src/frontend-samples/screen-space-effects-sample/ScreenSpaceEffectsUI.tsx

```diff
@@ -134,15 +143,15 @@ export default class ScreenSpaceEffectsUI extends React.Component<UIProps, UISta
         <span>Lens Distortion</span>
         <Toggle isOn={this.state.enableLensDistortion} onChange={(enableLensDistortion) => this.setState({ enableLensDistortion })} />
         <span>Lens Angle</span>
-        <input type="range" min="90" max="160" step="5" value={this.state.lensAngle} onInput={this._onChangeLensAngle} disabled={!this.state.enableLensDistortion}></input>
+        <Slider showMinMax={true} min={lensAngleMin} max={lensAngleMax} step={5} values={[lensAngleValue]} disabled={!this.state.enableLensDistortion} onUpdate={this._onUpdateLensAngle} />
         {this.createSlider("Strength", this.state.effectsConfig.lensDistortion.strength, 0, 1, 0.05, "enableLensDistortion", (config, val) => config.lensDistortion.strength = val)}
         {this.createSlider("Cylindrical Ratio", this.state.effectsConfig.lensDistortion.cylindricalRatio, 0, 1, 0.05, "enableLensDistortion", (config, val) => config.lensDistortion.cylindricalRatio = val)}
       </div>
     );
   }
 
   public render() {
-    const instructions = "Use the drop-down below to select which effect is applied to the viewport.";
+    const instructions = "Use the toggles below to select which effect is applied to the viewport.  Move the sliders to control those effects.";
```

Comment:
/s/effect is/effects are

Reply:
After reading the instructions today, I regret that second sentence and have removed it.